baremaps-postgres/src/main/java/com/baremaps/postgres/jdbc/CopyWriter.java

```diff
@@ -306,6 +308,16 @@ public void writeHstore(Map<String, String> value) throws IOException {
     nullableWriter(CopyWriter::hstoreWriter).write(data, value);
   }
 
+  /**
+   * Writes a jsonb array
+   *
+   * @param value
+   * @throws IOException
+   */
+  public void writeJsonb(Map<String, String> value) throws IOException {
```

Comment:
Interesting, can we simply write the JSON String in the DataOutputStream? If thats the case, I would simply change the signature of the writeJsonb method as follow:

```java
public void writeJsonb(String json) throws IOException {
```

In my understanding, this may allow us to support nested json structures such as the following one, which is not possible with a Map:

```json
{
  "number": 1,
  "string": "string",
  "array": [1,2,3,4],
  "object": {"key": "value"}
}
```

This may also enable the user to choose between different serialization and deserialization libraries (e.g. Gson, Jackson, etc.).

Would that be possible? What do you think?

Reply:
I was not sure if I could change the api so I kept  the same structure but I think it would be better to be given the json directly as you suggested.

Reply:
It should be pretty simple. Will look into it tomorrow